tests/backend/base/HttpTestCase.php

```diff
@@ -33,4 +43,35 @@ protected function enablePermissions(array $permissionsToEnable): void
     {
         $this->client->setServerParameter(SetHttpTestPermissionsListener::X_DPLAN_TEST_PERMISSIONS, implode(',', $permissionsToEnable));
     }
+
+    /**
+     * This method is essential for simulating an authenticated user in tests.
+     * Initializes a user for authentication by creating a JWT token,
+     * setting it in a JWTUserToken, and then setting this token in the token storage.
+     */
+    protected function initializeUser(User $user): string
```

Comment:
Most of this new methods is already done in `AbstractApiTest` and should be reused